libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Preds/CertState.hs

```diff
@@ -286,12 +330,19 @@ mainD seed = defaultMain $ testIO "Testing DState Stage" (demoD Interactive seed
 
 -- ===============================================
 
+tell :: String -> (Subst era -> a) -> (Subst era -> a)
+tell s f (Subst m) =
```

Comment:
`tell` is a name of a very popular function in `transformers` library. I'd suggest to rename it in order to avoid confusion.

Reply:
Actually that was a debugging function that is no longer needed.